src/utils/ipc/server.ts

```diff
@@ -36,6 +36,20 @@ export const createIpcServer = async () => {
 	const pipePath = getPipePath(process.pid);
 	await fs.promises.mkdir(tmpdir, { recursive: true });
 
+	/**
+	 * Fix #457 (https://github.com/privatenumber/tsx/issues/457)
+	 *
+	 * Avoid the error "EADDRINUSE: address already in use"
+	 *
+	 * If the pipe file already exists, it means that the previous process has been closed abnormally.
+	 *
+	 * We can safely delete the pipe file, the previous process must has been closed,
+	 * as pid is unique at the same.
+	 */
+	if (fs.existsSync(pipePath)) {
```

Comment:
This should be async. You can copy this over for now, and I'll abstract it out into a util later:
https://github.com/privatenumber/tsx/blob/985bbb8cff1f750ad02e299874e542b6f63495ef/src/esm/package-json.ts#L11-L14

Reply:
Good idea!